packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts

```diff
@@ -12,8 +14,23 @@ export const getCryptoCurrency = (state: RootState) =>
 export const getFiatCurrency = (state: RootState) =>
   state.components.simpleBuy.fiatCurrency || state.preferences.sbFiatCurrency
 
-export const getDefaultMethod = (state: RootState) =>
-  state.components.simpleBuy.defaultMethod
+export const getDefaultPaymentMethod = (state: RootState) => {
+  const ordersR = getSBOrders(state)
+  const sbMethodsR = getSBPaymentMethods(state)
+
+  const transform = (
+    orders: ExtractSuccess<typeof ordersR>,
+    sbMethods: ExtractSuccess<typeof sbMethodsR>
+  ) => {
+    const { paymentType: type, inputCurrency } = orders[0]
```

Comment:
is there a chance to have orders as `[]` ?

Reply:
Nice catch.